src/main/java/io/github/fusionflux/portalcubed/content/prop/P1Prop.java

```diff
@@ -0,0 +1,31 @@
+package io.github.fusionflux.portalcubed.content.prop;
+
+import io.github.fusionflux.portalcubed.content.PortalCubedDamageSources;
+import net.minecraft.util.Mth;
+import net.minecraft.world.damagesource.DamageSource;
+import net.minecraft.world.entity.EntitySelector;
+import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.level.Level;
+
+public class P1Prop extends Prop {
+	private static float MAX_FALL_DAMAGE = 30;
+	private static float FALL_DAMAGE_PER_BLOCK = 2 * 1.5f;
```

Comment:
Damage cap is arbitrary limit to prevent use against high-health mobs, fall damage per block is set so that it takes roughly the same fall distance to kill a player with no armor as it does in portal 1.  

The cap should also be 60 instead of 30, assuming its in half hearts.